### PR TITLE
Feature final capstone project for PR

### PR DESCRIPTION
**Pull request by supreeti**
This is my 1st Capstone project :- A Conference website for the upcoming event in 2023.
This website is responsive in nature using media queries with mobile first approach and desktop view .
Contains 2 pages a. Home page b. About page which can be navigated through homepage menu link.
Used :-1. HTML,CSS,
2.Javascript Array to store data as object and to dynamically fetch Speakers in home page using event listeners.[[]](https://github.com/supreeti/MYCAPSTONE/pull/1#issue-1585791050)(url)

### CODE REVIEW

**Comment by supreeti** [comment]
I made the required changes as suggested including renaming by branch from edulearn to Feature_Final_capstone_project. Please consider

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @supreeti , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Good job working on the presentation video and highlighting the piece of code you are proud of as requested by the previous reviewers but please kindly re-record and ensure that the time does not exceed 5 minutes.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.css [hunk 1]
```diff
@@ -0,0 +1,1147 @@
...
+  box-sizing: border-box;
+  font-family: "Lato", Helvetica, sans-serif;
+}
+
+.navbarimg {
+  margin-left: 20px;
+  margin-top: 15px;
+  z-index: 100;
+  position: absolute;
+}
```
Comment (AlphaNtihinduka) [inline index.css:13]: - [ ] [OPTIONAL] You have done a great job ensuring the design and implementation of the hamburger menu 👍 👏 . However, we should stick to the original design as much as possible ☺️ . Therefore I suggest that you change the color of the hamburger to match the original design.

Original design 😎 
image

Your design 😎 
HUMBER

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @supreeti 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- [ ] [OPTIONAL] Kindly ensure that once you make a pull request on dashboard, you should not go ahead to make any further changes. This gives a chance to the code reviewer to have a smoth review process.
- [ ] [OPTIONAL] Please ensure that when you create a new PR to ensure that the old PR is mentioned in the new PR description. This enables the reviewer to refer the reviews performed by previous reviewers.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**